Pass4Win/Main.cs

```diff
@@ -429,6 +420,7 @@ public static string EncryptConfig(string password, string salt)
         /// <returns></returns>
         public static string DecryptConfig(string cipher, string salt)
         {
+            if (string.IsNullOrEmpty(cipher)) return string.Empty;
```

Comment:
first time app runs cipher is null, this caused an unhandled exception and app crash